include/fastdds/dds/core/policy/ParameterTypes.hpp

```diff
@@ -165,22 +166,83 @@ class Parameter_t
     }
 
     /**
-     * Virtual method used to add the parameter to a CDRMessage_t message.
+     * Method used to add the parameter information to a CDRMessage_t message.
      * @param[in,out] msg Pointer to the message where the parameter should be added.
```

Comment:
Doxydoc for parameter `parameter` missing
Name of second parameter has is `cdr_message`, not `msg`